platforms/core-runtime/build-option/src/test/groovy/org/gradle/internal/buildoption/DefaultInternalOptionsTest.groovy

```diff
@@ -34,15 +34,15 @@ class DefaultInternalOptionsTest extends Specification {
         value1.explicit
 
         def value2 = options.getOption(new InternalFlag("prop2", false))
-        value2.get()
+        !value2.get()
         value2.explicit
 
         def value3 = options.getOption(new InternalFlag("prop3", true))
         !value3.get()
         value3.explicit
 
         def value4 = options.getOption(new InternalFlag("prop4", false))
```

Comment:
💭 

Since the default values are not relevant in this test (all options use explicit values), I would use single-arg constructor instead. 

(took me a bit to understand whether there was difference between line 40 and 44 - the only difference is in the explicitly assigned values